app/src/main/java/com/zeapo/pwdstore/pwgenxkpwd/Dictionary.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2014-2020 The Android Password Store Authors. All Rights Reserved.
+ * SPDX-License-Identifier: GPL-3.0-only
+ */
+package com.zeapo.pwdstore.pwgenxkpwd
+
+import android.content.Context
+import android.net.Uri
+import android.text.TextUtils
+import androidx.preference.PreferenceManager
+import com.zeapo.pwdstore.R
+import java.util.ArrayList
+import java.util.HashMap
+
+class Dictionary(context: Context) {
+    val words: HashMap<Int, ArrayList<String>>
+
+    init {
+        words = HashMap()
```

Comment:
I believe initialising this with the declaration is the same as doing it here.

Reply:
Could you please clarify: are you suggesting to replace `init {...` with an explicit constructor?

Reply:
No, just move the `= HashMap()` where `val words: HashMap<Int, ArrayList<String>>` is declared.